stories/components/color-picker/__stories__/docs-examples/input_validation.tsx

```diff
@@ -6,7 +6,7 @@ import { Flex } from 'intergalactic/flex-box';
 const Demo = () => {
   const [state, setState] = React.useState<'normal' | 'invalid'>('normal');
 
-  const onChange = (value) => {
+  const onChange = (value: any) => {
```

Comment:
`any` doesn't look typesafe here. Is there any other way to narrower type?

Reply:
this is not a task about 'fix types in color-picker'.
moreover, this is core problem with our typings, that's why I used `any` here